_data/engine-cli/docker_login.yaml

```diff
@@ -61,7 +61,7 @@ examples: |-
   `docker login` requires user to use `sudo` or be `root`, except when:
 
   1.  connecting to a remote daemon, such as a `docker-machine` provisioned `docker engine`.
-  2.  user is added to the `docker` group.  This will impact the security of your system; the `docker` group is `root` equivalent.  See [Docker Daemon Attack Surface](https://docs.docker.com/engine/security/security/#docker-daemon-attack-surface) for details.
+  2.  user is added to the `docker` group.  This will impact the security of your system; the `docker` group is `root` equivalent.  See [Docker Daemon Attack Surface](https://docs.docker.com/engine/security/#docker-daemon-attack-surface) for details.
```

Comment:
Manually edited this one, but corresponding PR is up in https://github.com/docker/cli/pull/2757